m3-sys/llvm/llvm9bindings/src/m3makefile

```diff
@@ -122,6 +125,34 @@ if equal (OS_TYPE, "POSIX")
   import_lib("dl",LIB_DIR)
   import_lib("stdc++",LIB_DIR)
   import_lib("z",LIB_DIR)
+else
+  % Both Windows amd64 and Windows X86
```

Comment:
nit: Suggest remove the comment. It is just for Windows. All architectures. No need to say.

Reply:
} Nit: It is just for Windows. No need to list all architectures.
I think about this... Why not?

Reply:
} Maybe no need to say "Windows".
Will be total delete? I am not author of first comment state...

Reply:
Fixed:
https://github.com/VictorMiasnikov/cm3/commit/19cfaed13c43fe431116b10811bac3ec71a35df1